deploy/olm-catalog/portworx/portworx.package.yaml

```diff
@@ -1,7 +1,7 @@
 packageName: portworx-certified
 channels:
 - name: alpha
-  currentCSV: portworx-operator.v24.1.1
+  currentCSV: portworx-operator.v24.1.2-drw
```

Comment:
@nikita-bhatia , I believe this should be changed into `portworx-operator.v24.1.3` or when the release is happening. 
Since this is part of olm, can you suggest what should be the value in the PR